qt/widgets/common/src/DataProcessorUI/QDataProcessorWidget.cpp

```diff
@@ -188,6 +188,9 @@ void QDataProcessorWidget::showTable(
   connect(m_model.get(),
           SIGNAL(dataChanged(const QModelIndex &, const QModelIndex &)), this,
           SLOT(rowDataUpdated(const QModelIndex &, const QModelIndex &)));
+  connect(m_model.get(),
+          SIGNAL(dataChanged(const QModelIndex &, const QModelIndex &)), this,
+          SIGNAL(dataChanged(const QModelIndex &, const QModelIndex &)));
```

Comment:
this confused me a little. I assume that there is a dataChanged in both the model and its self

Reply:
This is to pass a signal out of the c++ data processor widget to the python so the python view can do something with it. The signal gets passed via the view as the view is the only bit of the data processor widget the sans Gui really uses.

Reply:
can we have a issue to make this more MVP? Also can we have a brief comment to explain this for the future